test/harry/main/org/apache/cassandra/harry/dsl/HistoryBuilder.java

```diff
@@ -540,6 +541,15 @@ public void validateAll(Model model)
         }
     }
 
+    public void validateAll(Model model, Function<Long, List<Query>> queries)
```

Comment:
nit: Anybody using this?

Reply:
Not at the moment, but this still is a useful method, I have a follow-up patch that would use this (one that adds selection to Harry queries.